cmd/statusd/utils.go

```diff
@@ -972,7 +959,7 @@ func testCompleteMultipleQueuedTransactions(t *testing.T) bool {
 
 		time.Sleep(1 * time.Second) // make sure that tx complete signal propagates
```

Comment:
I don't think so. If you have figured out a better way to ensure that the node is in sync, this line should be fixed as well. But I think it's not in scope of this PR.